pageserver/src/walredo/apply_neon.rs

```diff
@@ -236,20 +234,17 @@ pub(crate) fn apply_in_neon(
                 LittleEndian::write_u32(&mut page[memberoff..memberoff + 4], member.xid);
             }
         }
-        NeonWalRecord::AuxFile { file_path, content } => {
-            let mut dir = AuxFilesDirectory::des(page)?;
-            dir.upsert(file_path.clone(), content.clone());
-
-            page.clear();
-            let mut writer = page.writer();
-            dir.ser_into(&mut writer)?;
+        NeonWalRecord::AuxFile { .. } => {
```

Comment:
why do we still keep it declared? it's an internal format that never hits the disk, no?

Reply:
It's stored in old layer files and I assume it will be decoded on the read path

Reply:
ahh wait, yes, it's never stored to the disk, but I didn't examine the read/write path carefully; maybe we can remove it, but I'd like to do it when it's safe to do so.